python/tskit/trees.py

```diff
@@ -3812,6 +3813,10 @@ def allele_frequency_spectrum(
         "node"
             Not supported for this method (raises a ValueError).
 
+        The method does *not* currently count the number of fixed alleles
```

Comment:
This is overstating it:
```python
ts = msprime.simulate(6, random_seed=2)
print(ts.draw_text())
afs = ts.allele_frequency_spectrum([[0, 2]], mode="branch", polarised=True)                                                  
print(afs)
```
gives

```
              10      ┊  
    ┊   ┏━━┻━━┓   ┊  
1.06┊   ┃     9   ┊  
    ┊   ┃   ┏━┻┓  ┊  
0.80┊   8   ┃  ┃  ┊  
    ┊ ┏━┻┓  ┃  ┃  ┊  
0.61┊ ┃  7  ┃  ┃  ┊  
    ┊ ┃ ┏┻┓ ┃  ┃  ┊  
0.08┊ ┃ ┃ ┃ ┃  6  ┊  
    ┊ ┃ ┃ ┃ ┃ ┏┻┓ ┊  
0.00┊ 3 0 2 1 4 5 ┊  
  0.00          1.00 

[[3.45905848 1.22532517 0.90607714]]

```
It's only stuff above the MRCA of **all** samples that's ignored, so the final entry will have values when the sample sets are a subset of all samples.

Reply:
Good point; I've fixed this.